LayoutTests/platform/glib/TestExpectations

```diff
@@ -2040,6 +2033,9 @@ webrtc/audio-sframe.html [ Skip ]
 webrtc/sframe-test-vectors.html [ Skip ]
 webrtc/sframe-transform-buffer-source.html [ Skip ]
 webrtc/video-sframe.html [ Skip ]
+webkit.org/b/219825 [ Release ] webrtc/sframe-keys.html [ Skip ]
+webkit.org/b/219825 [ Debug ] webrtc/sframe-keys.html [ Skip ]
```

Comment:
Why do we need the Release/Debug tags here? could it be just one line without any tag?